frontend/src/components/teacherTaskViewer/index.js

```diff
@@ -43,27 +120,140 @@ export default class TeacherTaskViewer extends React.Component {
 
         return (
             <Grid container style={{position: "absolute", width: "100%", height: "90%"}}>
-                <Grid xs={2} component={Paper} item style={{backgroundColor: "#f7f7f7"}}>
-                    <Box m={2}>
-                        <Card variant="outlined" style={{padding: 8}}>
-                            <Box textAlign="left" p={2}>
-                                <Typography variant="h6">Submissions</Typography>
+                <Grid xs={2} component={Paper} item style={{backgroundColor: "#f7f7f7", height: "93.5%", overflow: "scroll"}}>
+
+
+                    <Box m={1} >
+                        <ExpansionPanel style={{height: "50%"}}>
+                            <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />} aria-controls="panel1a-content" id="panel1a-header" >
+                                <Typography variant="h6" >Task Overview</Typography>
+                            </ExpansionPanelSummary>
+
+                            <ExpansionPanelDetails>
+                                    <Box textAlign="center">
+                                        <TextField
+                                        id="name"
+                                            label="Name"
+                                            variant="outlined"
+                                            size="small"
+                                            margin="dense"
+                                            fullWidth
+                                        />
+
+                                        <TextField
+                                            id="desc"
+                                            label="Description"
+                                            variant="outlined"
+                                            size="small"
+                                            margin="dense"
+                                            multiline
+                                            rows={8}
+                                            fullWidth                                  
+                                        />
+                                    </Box>
+                            </ExpansionPanelDetails>
+
+                        </ExpansionPanel>
+                    </Box>
+
+
+                    <Box m={1} >
+                        <ExpansionPanel style={{ height: "50%" }}>
```

Comment:
We need to move this kind of stuff into makeStyles or equivalent at some point, we can let it pass for now though